.github/CODEOWNERS

```diff
@@ -112,10 +112,11 @@ rush.json       @iTwin/itwinjs-core-admins
 /core/webgl-compatibility       @iTwin/itwinjs-core-display @Ellord207
 
 /docs   @iTwin/itwinjs-core
-/docs/getting-started @iTwin/itwinjs-core-admins @williamkbentley
-/docs/config          @iTwin/itwinjs-core-admins @williamkbentley
+/docs/getting-started @iTwin/itwinjs-core-admins @anmolshres98 @hl662 @DanRod1999
```

Comment:
Mike B is still mentioned in this file and it seems to be throwing an error. Can we fix int this PR?

Reply:
Yep

Reply:
Fixed